OREData/ored/portfolio/asianoption.cpp

```diff
@@ -132,23 +125,18 @@
                  if (assetClassUnderlying_ == AssetClass::EQ)
                      indexName = "EQ-" + indexName;
              }
-             for (QuantLib::Date observationDate : observationDates) {
+
+             std::vector<Real> fixingValues;
+             for (const Date& observationDate : observationDates) {
                  // TODO: Verify. Should today be read too? a enforcesTodaysHistoricFixings() be used? 
-                 if (observationDate < today ||
-                     (observationDate == today && Settings::instance().enforcesTodaysHistoricFixings())) {
```

Comment:
this feels like a loss of functionality, should `enforcesTodaysHistoricFixings` not apply here?

Reply:
This, too, I wrote. I've compared against other places that compare a fixing date vs the reference date/today and it's not common to see this (although it's done in a few places). Instead, we'll rely on the `DiscreteAsianOption` class' filtering of the dates when the running accumulator values are calculated. I haven't noticed any problems after doing this change.